src/App.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+import { Component } from 'react';
+import classNames from 'classnames';
```

Comment:
You can rename `classnames` to `cn` for abbreviation

Reply:
+